include/ensmallen_bits/sdp/primal_dual_impl.hpp

```diff
@@ -33,48 +33,7 @@
 #include "lin_alg.hpp"
 
 namespace ens {
-
-template<typename DeprecatedSDPType>
-ens_deprecated
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
-    const DeprecatedSDPType& sdp) :
-    deprecatedSDP(sdp),
-    initialX(arma::eye<arma::mat>(sdp.N(), sdp.N())),
-    initialYSparse(arma::ones<arma::vec>(sdp.NumSparseConstraints())),
-    initialYDense(arma::ones<arma::vec>(sdp.NumDenseConstraints())),
-    initialZ(arma::eye<arma::mat>(sdp.N(), sdp.N())),
-    maxIterations(1000),
-    tau(0.99),
-    normXzTol(1e-7),
-    primalInfeasTol(1e-7),
-    dualInfeasTol(1e-7)
-{ /* Nothing to do. */ }
-
-template<typename DeprecatedSDPType>
-ens_deprecated
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
-    const DeprecatedSDPType& sdp,
-    const arma::mat& initialX,
-    const arma::vec& initialYSparse,
-    const arma::vec& initialYDense,
-    const arma::mat& initialZ) :
-    deprecatedSDP(sdp),
-    initialX(initialX),
-    initialYSparse(initialYSparse),
-    initialYDense(initialYDense),
-    initialZ(initialZ),
-    maxIterations(1000),
-    tau(0.99),
-    normXzTol(1e-7),
-    primalInfeasTol(1e-7),
-    dualInfeasTol(1e-7)
-{
-  // Nothing to do.
-}
-
-template<typename DeprecatedSDPType>
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
-    const size_t maxIterations,
+inline PrimalDualSolver::PrimalDualSolver(const size_t maxIterations,
```

Comment:
I'm sorry, I guess I should have made this clear, you have to align all the parameters, as you can see from our style guidelines [here](https://github.com/mlpack/mlpack/wiki/DesignGuidelines#method-declarations)

Reply:
I have looked over the reference provided and made updates inline with it.